include/fmt/ranges.h

```diff
@@ -696,18 +696,18 @@ auto join(It begin, Sentinel end, string_view sep) -> join_view<It, Sentinel> {
  *     fmt::print("{:02}", fmt::join(v, ", "));
  *     // Output: 01, 02, 03
  */
-template <typename Range>
+template <typename Range, FMT_ENABLE_IF(!fmt::is_tuple_like<Range>::value)>
```

Comment:
Why qualify this with `fmt::`? I don't think ADL applies here.

Reply:
You are right.
Removed ``fmt::``.